db/system_distributed_keyspace.cc

```diff
@@ -40,6 +44,9 @@
 
 namespace db {
 
+thread_local data_type stream_tuple_type = tuple_type_impl::get_instance({long_type, long_type});
+thread_local data_type streams_set_type = set_type_impl::get_instance(stream_tuple_type, true);
```

Comment:
Add cdc_ prefix to make it clear to a non-cdc-oriented reader.

Reply:
Done